tensorboard/webapp/plugins/npmi/views/main/main_component.scss

```diff
@@ -44,3 +43,31 @@ limitations under the License.
   padding: 10px;
   width: calc(100% - 20px);
 }
+
+.violin-filters {
+  min-height: 0px;
```

Comment:
This is not a problem yet, but later there will be filters added. Once this happens, this can overflow, and if I dont have this min-height, for some reason, the wrong container overflows.